pkg/kubelet/network/cni/cni.go

```diff
@@ -53,14 +59,18 @@ type cniNetwork struct {
 
 func probeNetworkPluginsWithVendorCNIDirPrefix(pluginDir, vendorCNIDirPrefix string) []network.NetworkPlugin {
 	configList := make([]network.NetworkPlugin, 0)
-	network, err := getDefaultCNINetwork(pluginDir, vendorCNIDirPrefix)
-	if err != nil {
-		return configList
-	}
-	return append(configList, &cniNetworkPlugin{
-		defaultNetwork: network,
+	plugin := &cniNetworkPlugin{
+		defaultNetwork: nil,
```

Comment:
Accidentally deleted previous comment, but at a glance this looks like it could result in nil pointer errors in calls to the NetworkPlugin methods. So maybe just validating this assumption is safe via tests, or maybe a nullPlugin which logs/returns errors on those same calls until a real plugin config is picked up .(or my reading of this is off).